scripts/xtensa-build-all.sh

```diff
@@ -40,7 +40,7 @@ usage: $0 [options] platform(s)
        -u Force UP ARCH
        -d Enable debug build
        -c Interactive menuconfig
-       -o copies the file argument from src/arch/xtensa/configs/override/$arg.config
+       -o arg copies src/arch/xtensa/configs/override/<arg>.config
 	  to the build directory after invoking CMake and before Make.
        -k Configure rimage to use a non-default \${RIMAGE_PRIVATE_KEY}
```

Comment:
help option is not very clear here. Maybe, `-o arg, copies the file argument from ....`. In your sentence it is not very clear what/where it is copied.

Reply:
Thanks for the comma suggestion, I agree it would help a lot. I'm not sure any other change would be required then because the sentence is: "copies X to the build directory".

Reply:
@marc-hb oh, i stopped reading after first line. You are right. So I'm fine after adding the comma.